degrees_at_center_pulse_width:
- 0
- -45
- 135
- 0
- -45
- 135
- 0
- -45
- 135
- 0
- -45
- 135
leg_code:
- FLH
- FLU
- FLL
- FRH
- FRU
- FRL
- BLH
- BLU
- BLL
- BRH
- BRU
- BRL
max_angle_degrees:
- 20
- 0
- 157.5
- 20
- 0
- 157.5
- 20
- 0
- 157.5
- 20
- 0
- 157.5
min_angle_degrees:
- -20
- -67.5
- 112.5
- -20
- -67.5
- 112.5
- -20
- -67.5
- 112.5
- -20
- -67.5
- 112.5
pulse_width_per_degree:
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
- 10.0
zero_degrees_pulse_width:
- 1500
- 1500
- 1500
- 1500
- 1500
- 1500
- 1500
- 1500
- 1500
- 1500
- 1500
- 1500
